Pinta.Tools/Tools/ColorPickerTool.cs

```diff
@@ -172,7 +173,7 @@ private ColorBgra[] GetPixelsFromPoint (Document document, PointI point)
 			for (var j = rect.Top; j <= rect.Bottom; j++)
 				pixels.Add (GetPixel (document, i, j));
 
-		return pixels.ToArray ();
+		return pixels.ToImmutableArray ();
```

Comment:
As a bonus change, since we know what the size of this list will be (based on the size of `rect`) this could be an ImmutableArrayBuilder with the appropriate capacity